.travis.yml

```diff
@@ -3,8 +3,8 @@ language: android
 android:
   components:
     - tools
-    - build-tools-27.0.3
-    - android-27
+    - build-tools-28.0.2
```

Comment:
@voltazor according with travis, we don't need to change build tools version since we didn't touch it, just the android version.